Gemfile

```diff
@@ -141,3 +141,5 @@ gem 'active_model_serializers', github: 'rails-api/active_model_serializers', re
 gem 'aws-sdk', '~> 2'
 
 gem 'rubocop', require: false, group: [:development, :staging]
+# Reduce volume of production logs
+gem 'lograge'
```

Comment:
The installation instructions on https://github.com/roidrage/lograge suggest adding `gem "lograge"` to the Gemfile and then "Enable it for the relevant environments, e.g. production" by adding the `enabled` flag in the environment initializers. But if you know how to get this work with the Bundler group, please let me know and I can try it out